client/hosting-overview/components/plan-card.tsx

```diff
@@ -172,8 +173,12 @@ const PlanCard: FC = () => {
 	);
 	const isStaging = isStagingSite( site ?? undefined );
 	const isAtomic = useSelector( ( state: AppState ) => isAtomicSite( state, site?.ID ?? 0 ) );
+	const { hasSftpFeature } = useSelector( ( state: AppState ) => ( {
```

Comment:
I'm basing the logic for atomic eligibility on how we handle it for displaying the "activate hosting features" vs the "upgrade" button on the Hosting Settings screen. After testing a bunch this setup seems to work well.